src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -122,344 +117,32 @@ protected override void EndProcessing()
-        /// <param name="depth"></param>
-        /// <returns></returns>
-        private object ProcessEnumerable(IEnumerable enumerable, int depth)
-        {
-            List<object> result = new List<object>();
-
-            foreach (object o in enumerable)
-            {
-                result.Add(ProcessValue(o, depth + 1));
-            }
-
-            return result;
-        }
-
-        /// <summary>
-        /// Return an alternate representation of the specified aggregate object that serializes the same JSON, except
-        /// that any contained properties that cannot be evaluated are treated as having the value null.
-        ///
-        /// The result is a dictionary in which all public fields and public gettable properties of the original object
-        /// are represented.  If any exception occurs while retrieving the value of a field or property, that entity
-        /// is included in the output dictionary with a value of null.
-        /// </summary>
-        /// <param name="o"></param>
-        /// <param name="depth"></param>
-        /// <returns></returns>
-        private object ProcessCustomObject<T>(object o, int depth)
-        {
-            Dictionary<string, object> result = new Dictionary<string, object>();
-            Type t = o.GetType();
-
-            foreach (FieldInfo info in t.GetFields(BindingFlags.Public | BindingFlags.Instance))
-            {
-                if (!info.IsDefined(typeof(T), true))
-                {
-                    object value;
-                    try
-                    {
-                        value = info.GetValue(o);
-                    }
-                    catch (Exception)
-                    {
-                        value = null;
-                    }
-
-                    result.Add(info.Name, ProcessValue(value, depth + 1));
-                }
-            }
-
-            foreach (PropertyInfo info2 in t.GetProperties(BindingFlags.Public | BindingFlags.Instance))
-            {
-                if (!info2.IsDefined(typeof(T), true))
-                {
-                    MethodInfo getMethod = info2.GetGetMethod();
-                    if ((getMethod != null) && (getMethod.GetParameters().Length <= 0))
-                    {
-                        object value;
-                        try
-                        {
-                            value = getMethod.Invoke(o, new object[0]);
-                        }
-                        catch (Exception)
-                        {
-                            value = null;
-                        }
-
-                        result.Add(info2.Name, ProcessValue(value, depth + 1));
-                    }
-                }
-            }
-
-            return result;
-        }
-
-        /// <summary>
-        /// Exception used for Stopping.
-        /// </summary>
-        private class StoppingException : System.Exception { }
+        #endregion overrides
```

Comment:
Is it useful in modern editors? I'd remove all regions.

Reply:
It's useful when I want to focus on part of the functionality/code. For example, when I'm working on the `ConvertToJson`, I can collapse the `ConvertFromJson` region, and it makes it easy for me to look at `ConvertToJsonContext` and the use of it.